README.md

```diff
@@ -27,25 +28,25 @@ Spark K8s Operator is built using Gradle.
 To build, run:
 
 ```bash
-$ ./gradlew build -x test
+./gradlew build -x test
 ```
 
 ## Running Tests
 
 ```bash
-$ ./gradlew build
+./gradlew build
 ```
 
 ## Build Docker Image
 
 ```bash
-$ ./gradlew buildDockerImage
+./gradlew buildDockerImage
 ```
 
-## Install Helm Chart
+## Install Helm Chart from the source code
```

Comment:
The linter detected that we had the duplicated section title at line 12 and line 45.